pkg/client/client_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 The Cluster Monitoring Operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/openshift/cluster-monitoring-operator/pkg/manifests"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func TestCreateOrUpdateClusterRoleBinding(t *testing.T) {
```

Comment:
could we add a test that asserts the role binding is not changed if annotations are added?

Reply:
and that annotations are preserved.

Reply:
added